test/backend/routes.spec.js

```diff
@@ -292,9 +295,10 @@ describe('Authentication - ', () => {
             GET('settings')
             .then(res => {
                 assert.equal(res.status, 200);
-                done();
-            }).catch(done);
-        });
+                return;
+            });
+            done();
+        }).catch(done);
```

Comment:
https://github.com/plotly/plotly-database-connector/pull/180/files/e7034e717d8602fa95158af34447b5a8d4e7f712..7204ec66554f53e5732be3d1da64fcd020c0323d#r143551641 and https://github.com/plotly/plotly-database-connector/pull/180/files/e7034e717d8602fa95158af34447b5a8d4e7f712..7204ec66554f53e5732be3d1da64fcd020c0323d#r143551795 apply here as well